pallets/stellar-relay/src/tests.rs

```diff
@@ -534,13 +541,14 @@ fn verify_signature_works_for_mock_message() {
 fn update_tier_1_validator_store_old_organization_and_validator_and_block_height_works() {
 	run_test(|_, _, _| {
 		let organization = Organization { id: 0, name: Default::default() };
+
 		let validator = Validator {
 			name: Default::default(),
-			public_key: Default::default(),
+			public_key: BoundedVec::<u8, FieldLength>::try_from(vec![1u8; 128]).unwrap(),
```

Comment:
Why are you not using `Default::default()` anymore?

Reply:
I want explicitly specify the public key for validator.